codespell_lib/data/dictionary.txt

```diff
@@ -10532,6 +10532,7 @@ disscussing->discussing
 disscussion->discussion
 disscussions->discussions
 disshearteningly->dishearteningly
+dissimialrity->dissimilarity
 dissimiliarity->dissimilarity
```

Comment:
A few more variations
```suggestion
dissimialr->dissimilar
dissimialrity->dissimilarity
dissimiliar->dissimilar
dissimiliarity->dissimilarity
```

Reply:
I took all lines containing `->similar` and prefixed them with `dis`.